src/clj/rems/api/services/util.clj

```diff
@@ -4,18 +4,23 @@
             [rems.db.users :as users]))
 
 (defn forbidden-organization? [user-id organization]
+  ;; XXX: If called multiple times with the same user-id, queries database
+  ;;      again each time.
   (let [user (users/get-user user-id)
         not-owner? (not (contains? (roles/get-roles user-id) :owner))
```

Comment:
hmm, even though you pass in user-id explicitly, you fetch the roles from the db

Reply:
Hmm, not sure how to do it otherwise. Do you have a proposal?

Reply:
Ah, I forgot that the roles exist in the context, as well. Right, I'll have a look at that.

Reply:
Hmm, you also fetch the user details from the db...

I can't quite put my finger on what bothers me, but I kinda think that the API of this function should be either `(forbidden-organization? organization)` or `(forbidden-organization? user roles organization)` where `user` is a map of the user attributes, not just the user id... but now that I think about it, `(forbidden-organization? user-id organization)` is an ok API too. I guess I was just expecting something else.

Reply:
Done, I'll change it to `(forbidden-organization? organization)`.